docs/mp/grpc/server.adoc

```diff
@@ -270,7 +273,7 @@ public StreamObserver<RequestType> invoke(StreamObserver<ResponseType> observer)
 === Deploying Protobuf Services
 Whilst the examples above show how simple it is to write gRPC services with basic POJOs, there may be cases where there is a
 requirement to deploy services built the traditional way using gRPC Java generated classes or built as
-xref:../../se/grpc/server.adoc#_service_implementation[non-microprofile Helidon gRCP services].
+xref:../../se/grpc/server.adoc#_service_implementation[non-microprofile Helidon gRPC services].
```

Comment:
Should have mentioned this earlier...Ideally, use the `rootdir` attribute in the reference and avoid the relative reference.

Reply:
resolved across all docs